### PR TITLE
Theme file generator

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**

### CODE REVIEW

**Review by cb4ds — APPROVED** [review]

File: vignettes/themeBuilder_addin.Rmd [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  %\VignetteEncoding{UTF-8}
+---
+
+## Overview
+
+To help package users in creating valid theme configuration file
+
+## Add-in Launch
+
+The add-in can be lunched by one of two methods
```
Comment (agenius-jennifer) [inline vignettes/themeBuilder_addin.Rmd:21]: Spelling: "lunched" should be "launched"

File: vignettes/themeBuilder_addin.Rmd [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Add-in Layout
+
+- The add-in will open as web browser tab
+- Add-in UI consists of:
+  - Header
+  - Body
+  - Footer
+- Add-in header that consists of:
+  - Cancel/Done buttons: they are default add-ins buttons and their solo purpose is to close add-in window
```
Comment (agenius-jennifer) [inline vignettes/themeBuilder_addin.Rmd:36]: Spelling: "solo" should be "sole"

File: vignettes/themeBuilder_addin.Rmd [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## Colors Contrast Tab
+
+These variables allow to customize color used if contrast between a color and its background is under threshold. For example, it's used to choose text color written in bs4ValueBox with background defined by a status
+
+<center><img src="figures/theme_6.png" width="600px"></center>
+
+## Other Variables Tab
+
+This is an advanced method to add customize any more AdminLTE or Bootstrap 4 variable in periscope2 generated application theme.
```
Comment (agenius-jennifer) [inline vignettes/themeBuilder_addin.Rmd:84]: Is there supposed to be another word between "add customize"? Maybe "add or customize"?

File: vignettes/themeBuilder_addin.Rmd [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        toc_depth: 3
+vignette: >
+  %\VignetteIndexEntry{Theme Configuration Builder}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Overview
+
+To help package users in creating valid theme configuration file
```
Comment (agenius-jennifer) [inline vignettes/themeBuilder_addin.Rmd:17]: The period "." is missing at the end of this sentence, and similarly for many sentences throughout the vignette.

File: vignettes/themeBuilder_addin.Rmd [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- Label: "Download"
+
+## Downloaded File
+Name: periscope_style.yaml
+Format: Value are based on input configuration but output is similar to the following
+
+<center><img src="figures/theme_9.png" width="600px"></center>
+
+## Downloaded Configuration File Usage
+The generate file is used by putting it inside the generated app www folder where it will be located by default
```
Comment (agenius-jennifer) [inline vignettes/themeBuilder_addin.Rmd:105]: Spelling: "generate" should be "generated"

File: vignettes/themeBuilder_addin.Rmd (hunk 1 above)
Comment (agenius-mohammed-ali) [reply vignettes/themeBuilder_addin.Rmd:21]: done

File: vignettes/themeBuilder_addin.Rmd (hunk 2 above)
Comment (agenius-mohammed-ali) [reply vignettes/themeBuilder_addin.Rmd:36]: done

File: vignettes/themeBuilder_addin.Rmd (hunk 3 above)
Comment (agenius-mohammed-ali) [reply vignettes/themeBuilder_addin.Rmd:84]: done

File: vignettes/themeBuilder_addin.Rmd (hunk 4 above)
Comment (agenius-mohammed-ali) [reply vignettes/themeBuilder_addin.Rmd:17]: done, I will do more deep review for all articles before submitting it to CRAN

File: vignettes/themeBuilder_addin.Rmd (hunk 5 above)
Comment (agenius-mohammed-ali) [reply vignettes/themeBuilder_addin.Rmd:105]: done

**Pull request merged**